spec/duration.html

```diff
@@ -302,7 +302,7 @@ <h1>Temporal.Duration.prototype.plus ( _other_ [ , _options_ ] )</h1>
         1. Let _duration_ be the *this* value.
         1. Perform ? RequireInternalSlot(_duration_, [[InitializedTemporalDuration]]).
         1. Set _other_ to ? ToLimitedTemporalDuration(_other_, « »).
-        1. Let _disambiguation_ be ? ToArithmeticTemporalDisambiguation(_options_).
+        1. Let _disambiguation_ be ? ToTemporalDisambiguation(_options_).
```

Comment:
Currently, `Duration.prototype.plus` doesn't accept `disambiguation: 'balance'`. So I think the PR code above is correct, at least according to the current [docs](https://tc39.es/proposal-temporal/docs/duration.html#plus) and polyfill.

![image](https://user-images.githubusercontent.com/277214/83671286-4c741500-a589-11ea-9282-faf46ce356fd.png)

That said, outside the scope of this PR I don't think this is a good design.  Seems like `plus` should support balancing like `minus` does, so that users won't be confused by the inconsistency. See #645.  Thanks for noticing this.

Reply:
Thanks for filing the issue, the PR in itself LGTM. Merging.